python/cudf_polars/cudf_polars/dsl/utils/aggregations.py

```diff
@@ -89,17 +89,21 @@ def decompose_single_agg(
     """
     agg = named_expr.value
     name = named_expr.name
-    if isinstance(agg, expr.UnaryFunction) and agg.name == "rank":
+    if isinstance(agg, expr.UnaryFunction) and agg.name in {
+        "rank",
+    }:
```

Comment:
Note: Not necessary in this PR, but I'm about to support `fill_null_with_strategy` and `cum_sum` in the next two PRs, so I'll end up making this change anyway.